web/src/stream_settings_components.ts

```diff
@@ -35,6 +35,19 @@ export function set_right_panel_title(sub: StreamSubscription): void {
     );
 }
 
+export function resize_stream_list_toggler_container(resize_window = true): void {
```

Comment:
I think these function for both streams and groups UI can be moved to `resize.ts` where we have other resize handlers.